docs/users-guide/13-embedding.md

```diff
@@ -0,0 +1,62 @@
+## Embedding Metabase in other Applications
+
+### Key Concepts
+
+#### Applications
+An important distinction to keep in mind is the difference between Metabase and the embedding application. The charts and dashboards you will be embedding live in the Metabase application, and will be embedded in your (i.e. the embedding application). 
+
+#### Parameters 
+Some dashboards, and questions have the ability to accept parameters. In dashboards, these are synnomous with dashboard filters. For example, if you have a dashboard with a filter on Publisher ID, this can be supplied as a filter when embedding, so that you could insert the dashboard filtered down to a specific Publisher ID.
```

Comment:
Remove comma: `Some dashboards, and questions`
Typo: `synnomous` should be `synonymous`
Possibly change "filter" in this spot to "parameter"?: `this can be supplied as a filter when embedding`